src/FirstSteps/RPG/Game.cs

```diff
@@ -32,38 +24,21 @@ public static void CreateHero()
             _hero = HeroesCreator.Create(name, race);
 
         }
-        public static void HandleCommand(string command)
+        public static void HandleAdventures(string command)
```

Comment:
`HandleCommand` was a good name because this method handles adventures and stats commands.